src/views/dataset/MultiSourceConfiguration.vue

```diff
@@ -104,16 +105,46 @@ export default class NewDataset extends Vue {
   get items() {
     return this.dimensions
       .filter(dim => dim.size > 0)
-      .map((dim: any) => ({ ...dim, key: `${dim.id}_${dim.source}` }));
+      .map((dim: IDimension) => {
+        let value = "";
+        const metadataID = this.dimensionTometadataId(dim.id);
+        switch (dim.source) {
+          case Sources.Filename:
+            if (this.collectedMetadata) {
+              const exampleValues = this.collectedMetadata.metadata[metadataID];
+              value = exampleValues.slice(0, 3).join(", ");
+              if (exampleValues.length > 3) {
+                value = String.prototype.concat(value, "...");
```

Comment:
value = value.concat(value, "...") ?

Reply:
This part of the code has been edited in another branch